### PR TITLE
[KOGITO-1648] Fix examples configuration and instructions

### PR DESCRIPTION
**Pull request by jstastny-cz**

### CODE REVIEW

File: kogito-business-rules-springboot/README.md [hunk 1]
```diff
@@ -85,88 +85,143 @@ You can take a look at the [OpenAPI definition](http://localhost:8080/docs/swagg
...
+where uuid is persons id and tuuid is task id.
 
-```
-{"person":{"name":"john","age":5,"adult":false},"id":"c59054b9-aa1d-4771-bc5e-40f8b32d3ff5","name":"ChildrenHandling"}
+It should return something similar to
+
+```json
+{
+    "person":{
+        "name":"john"
```
Comment (jomarko) [inline kogito-business-rules-springboot/README.md]: missing comma

File: kogito-business-rules-quarkus/README.md [hunk 2]
```diff
@@ -109,73 +130,122 @@ To make use of this application it is as simple as putting a sending request to
...
+where uuid is persons id and tuuid is task id.
 
-```
-{"person":{"name":"john","age":5,"adult":false},"id":"c59054b9-aa1d-4771-bc5e-40f8b32d3ff5","name":"ChildrenHandling"}
+It should return something similar to
+
+```json
+{
+    "person":{
+        "name":"john"
```
Comment (jomarko) [inline kogito-business-rules-quarkus/README.md]: missing comma

**Review by jomarko — COMMENTED** [review]
This is valuable contribution! Thank you @jstastny-cz

**Review by cristianonicolai — APPROVED** [review]

File: kogito-business-rules-quarkus/README.md (hunk 2 above)
Comment (jstastny-cz) [reply kogito-business-rules-quarkus/README.md]: I was wondering why the colon below is RED! fixing

Comment (jstastny-cz) [inline kogito-business-rules-quarkus/README.md]:
```suggestion
        "name":"john",
```

File: kogito-business-rules-springboot/README.md (hunk 1 above)
Comment (jstastny-cz) [reply kogito-business-rules-springboot/README.md]:
```suggestion
        "name":"john",
```

**Review by evacchi — APPROVED** [review]

**Review by ricardozanini — APPROVED** [review]

**Review by jomarko — APPROVED** [review]
once more, thank you for this PR @jstastny-cz

**Comment by ricardozanini** [comment]
Guys I thing this PR (https://github.com/kiegroup/kogito-examples/pull/168) should be merged first, then @jstastny-cz could do the final review/rebase. I've incorporated some of these changes as well.

**Review by tiagodolphine — APPROVED** [review]
@jstastny-cz great work, LGTM.

**Comment by tiagodolphine** [comment]
@jstastny-cz  there is a conflict to be solved.

**Comment by jstastny-cz** [comment]
@ricardozanini I am not sure if merging #168 first is the easier path. I did not change structure in this PR, so putting #168 on top of this one should be pretty straight forward. letme check locally ;-)

EDIT: You were right, let's merge #168 first, I'll deal with only 10 conflicts afterwards.

**Comment by ricardozanini** [comment]
@jstastny-cz I'll redo some of my work in the examples, we hopefully will merge that tomorrow :crossed_fingers:

**Comment by jstastny-cz** [comment]
rebased

**Pull request merged**